0.5/tests/test_validation.py

```diff
@@ -60,6 +60,11 @@ def pytest_generate_tests(metafunc):
     if "suite" in metafunc.fixturenames:
         suites: List[Schema] = []
         ids: List[str] = []
+        schema_store = {}
```

Comment:
Assuming we move to the extracted 0.6-dev branch in https://github.com/joshmoore/ngff/commit/878e15c550b4b71a3b0d7e690ce64dd91d963ca6 then we should move this separately to the v0.5 branch.